### PR TITLE
Adding default switch file

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

To be able to change one line in the source-code, re-deploy and disable a payment flow.

### tldr;

After the brilliant work done by @Ap0c with the switchable component, I am adding a helper with the default switches and pushing them into the common state. Then I am reading these switches from the common state whenever is necessary. 

## Next steps, for the future:
This work almost concludes the client side work required for the switchboard, next steps are deciding a place where to save the switches and reading them in the `switch.js` from that place, this logic should be located in the init function (which currently return the default switches). 

[**Trello Card**](https://trello.com)

## Changes

* Define default switches
* Push the switches to the common state
* Read the switches status form the common state and passing it to the different buttons whenever is necessary.

## Screenshots

If we would like to disable any of the payment flow, we go to the `helpers/switch.js` file and set it to `OFF` (eg. `directDebit`).
```
const defaultSwitches: Switches = {
  oneOffPaymentMethods: {
    stripe: 'ON',
    payPal: 'ON',
  },
  recurringPaymentMethods: {
    stripe: 'ON',
    payPal: 'ON',
    directDebit: 'OFF',
  },
};
```

Then we can see that the button is disabled on the checkout page:

switchboardexample

### CODE REVIEW

**Comment by svillafe** [comment]
Tested in `CODE` and is working ✅
